qemu/tests/win_virtio_driver_install_from_update.py

```diff
@@ -73,7 +73,18 @@ def start_wuauserv_service(session):
     # after uninstall
     if driver_name in ("viostor", "vioscsi"):
         time.sleep(120)
-    if not utils_misc.wait_for(lambda: not session.cmd_status(check_stat), 600, 0, 10):
+
+    driver_svc = params.get("driver_svc", driver_name)
```

Comment:
And how about using the dict in https://github.com/avocado-framework/avocado-vt/blob/master/virttest/utils_windows/virtio_win.py#L18?  Then there is no need to define 'driver_svc' parameter.

Reply:
Actually define 'driver_svc' parameter, seems be more easier.
But I preferred to accept this suggestion, as It will be convenient for us to track where the  'driver_svc' come from the first time.